src/core/ui.js

```diff
@@ -222,7 +224,7 @@ export const ui = {
     const id = `${currentOwner.id}-modal`;
     const headingId = `${id}-heading`;
     modal = hyperHTML`<div id='${id}' class='respec-modal removeOnSave' role='dialog'>
-      <h3 id="${headingId}">${title}</h3>
+      <h3 id="${headingId}">${closeButton}${title}</h3>
```

Comment:
I'm not sure of the accessibility implications of adding close button next to heading. It'll be announced as: "heading level 3 button close Search definitions"
Might be better to move this out of the heading and add a `aria-labelledby="${headingID}"` on `div[role='dialog']` above.

Reply:
Should the close button be at the same place?
![Screenshot from 2020-01-09 13-48-19](https://user-images.githubusercontent.com/44131389/72067166-2b740480-3308-11ea-9ab2-017099a0fcfd.png)
Also, shall I write it something like this:
```
<div id="${headingId}">${closeButton} <h3>${title}</h3></div>
```